### PR TITLE
Fix list all uniq container images

### PR DESCRIPTION
**Pull request by ms-choudhary**
Fix https://github.com/kubernetes/website/issues/27475

### CODE REVIEW

**Comment by sftim** [comment]
@ms-choudhary we prefer to [limit pull requests to a single localization](https://kubernetes.io/docs/contribute/new-content/overview/#languages-per-pr)

Could you revise this to change just the English page(s)? Other localization teams each have their own workflow for picking up changes from the upstream (English) content.

**Comment by ms-choudhary** [comment]
@sftim Done

**Comment by atoato88** [comment]
@ms-choudhary 
Could you remove `language/*` labels except for `language/en`?
Because current code includes just content for `en`.

**Comment by ms-choudhary** [comment]
>  Could you remove language/* labels except for language/en?
I don't seem to have access to update the labels actually. Might have been auto picked based on my first commit. Should I raise a new PR?

**Comment by atoato88** [comment]
@ms-choudhary 
In situation of this PR, you can refer [this comment of PR(#27993)](https://github.com/kubernetes/website/pull/27993#issuecomment-843137237).
That command will remove labels.

You can use [prow commands](https://prow.k8s.io/command-help) in some k8s repository for updating and collaborating ..., please see it :smile:

**Comment by ms-choudhary** [comment]
thanks @atoato88

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

File: content/en/docs/tasks/access-application-cluster/list-all-running-container-images.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ of Containers for each.
 
 - Fetch all Pods in all namespaces using `kubectl get pods --all-namespaces`
 - Format the output to include only the list of Container image names
-  using `-o jsonpath={..image}`.  This will recursively parse out the
+  using `-o jsonpath={.items[*].spec.containers[*].image}`.  This will recursively parse out the
   `image` field from the returned json.
```
Comment (joebowbeer) [inline content/en/docs/tasks/access-application-cluster/list-all-running-container-images.md:27]:
```suggestion
  using `-o jsonpath={.items[*].spec.containers[*].image}`
```

This no longer recursively parses out every image field.

File: content/en/docs/tasks/access-application-cluster/list-all-running-container-images.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ of Containers for each.
   - Use `uniq` to aggregate image counts
 
 ```shell
-kubectl get pods --all-namespaces -o jsonpath="{..image}" |\
+kubectl get pods --all-namespaces -o jsonpath="{.items[*].spec.containers[*].image}" |\
 tr -s '[[:space:]]' '\n' |\
 sort |\
 uniq -c
```
Comment (joebowbeer) [inline content/en/docs/tasks/access-application-cluster/list-all-running-container-images.md:39]: Delete lines 40-51 as the alternative is no longer relevant.

**Review by joebowbeer — COMMENTED** [review]
See https://github.com/kubernetes/website/issues/37061

**Pull request merged**